### PR TITLE
release: v0.1.1

### PR DESCRIPTION
**Pull request by simbleau**
Minor patch release to fix view-boxes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,12 +10,16 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
...
+## 0.1.1
+
+### Added
+
 - Make `util` module public and some minor doc fixes. [#12](https://github.com/linebender/vello_svg/pull/12)
 
 ### Changed
 
 - Updated `usvg` to 0.41
+- Disable `vello`'s default `wgpu` feature, and provide a `wgpu` passthrough feature to turn it back on. [#10](https://github.com/linebender/vello_svg/pull/10)
```
Comment (DJMcNab) [inline CHANGELOG.md:22]: This is a default feature, right?

**Review by DJMcNab — APPROVED** [review]
I have not confirmed that the new version number is correct, but it's good to be doing more regular releases.

**Comment by simbleau** [comment]
I don't want to do semver with this- I want to keep parity with Vello.

In other words, vello_svg 0.1.X pairs with Vello 0.1

**Review by DJMcNab — CHANGES_REQUESTED** [review]
> Semver also really only applies after 1.0

Rust (and, except for these cases, Linebender) uses a [modified definition](https://doc.rust-lang.org/cargo/reference/semver.html) of semver which makes semver apply before 1.0. cargo is implemented using this version of semver, and so this change has the possibility of breaking our users e.g. in CI.

In this case the breaking change is minor enough that I'm not immediately yanking 0.1.1. But I also don't want a  repeat of this before we have consensus that this is the direction we want to take.
If we do decide to follow this versioning scheme, we need to document that users SHOULD pin the version of `vello_svg` they depend on.

Rejecting to dismiss my prior approval

**Pull request merged**